rebasehelper/specfile.py

```diff
@@ -605,7 +529,7 @@ def set_extra_version(self, extra_version: Optional[str], version_changed: bool)
         # TODO: in some cases it might be necessary to modify Source0
 
     @saves
-    def set_tag(self, tag: str, value: str, preserve_macros: bool = False, subpackage: Optional[str] = None) -> None:
```

Comment:
Why don't we need the subpackage? What if we wanted to alter a tag in a subpackage, this wouldn't be possible after the removal, right?

Reply:
Feel free to adjust `set_tag()` to work with subpackages, but I don't think it's worth it. What would be the use-case?

Reply:
Makes sense, there is probably no point in having `set_tag` work with subpackages, at least for now